android/301Project/app/src/main/java/com/example/a301project/AddEditIngredientFragment.java

```diff
@@ -80,6 +80,25 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
         unitName = view.findViewById(R.id.edit_unit);
         categoryName = view.findViewById(R.id.edit_category);
 
+        // Category spinner
+        ArrayAdapter<CharSequence> categoryAdapter = ArrayAdapter.createFromResource(this.getContext(),
```

Comment:
Should this be a string instead of charsequence

Reply:
I believe it does not work with String from my short testing. I don't think it matters too much